src/app/components/hotkeys-page/hotkeys-page.component.html

```diff
@@ -0,0 +1,124 @@
+<div class="page"> 
```

Comment:
Since we're wrapping every template with this page div, it can be moved into app.template.html and wrap the router-outlet element with it, then remove the page class from components that have it on their root section.

Reply:
I noticed that some components use `<section>` not `<div>`

Reply:
Yes, should be the same.